actions/extractor/tools/autobuild.sh

```diff
@@ -4,8 +4,10 @@ set -eu
 
 DEFAULT_PATH_FILTERS=$(cat << END
 exclude:**/*
-include:.github/workflows/**/*.yml
-include:.github/workflows/**/*.yaml
+include:.github/workflows/*.yml
+include:.github/workflows/*.yaml
+include:.github/reusable_workflows/**/*.yml
+include:.github/reusable_workflows/**/*.yaml
```

Comment:
Is it worth adding a comment with some of the context from the Slack conversation about why we include these directories?

Reply:
See 661e9da.